networking/p2p/src/main/java/tech/pegasys/artemis/networking/p2p/mothra/rpc/RPCCodec.java

```diff
@@ -59,13 +52,7 @@ public static Bytes encode(Object request) {
    * @param message the bytes of the message to read
    * @return the payload, decoded
    */
-  public static RPCMessage decode(Bytes message) {
-    try {
-      byte[] body = message.toArrayUnsafe();
-      ObjectNode bodyNode = (ObjectNode) mapper.readTree(body);
-      return new RPCMessage(bodyNode.get("body"));
-    } catch (IOException e) {
-      throw new UncheckedIOException(e);
-    }
+  public static Bytes decode(Bytes message) {
+    return message;
```

Comment:
maybe decode it into the HELLO message?  since there will be other message types it could return a generic param and accept a class type...?